style.css

```diff
@@ -245,6 +223,84 @@ h3 {
   cursor: pointer;
 }
 
+/* The contact form */
+.form-content {
+  padding: 24px;
+}
+
+.form-content h2 {
+  color: #9093ff;
+  padding: 0;
+  font-size: 32px;
+  text-align: center;
+  margin-top: 50px;
+  margin-left: 20px;
+  font-weight: 500;
+}
+
+.form-content p {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f5f5ff;
+  margin-top: 24px;
+  margin-bottom: 40px;
+  width: 327px;
+  margin-left: 35px;
+  font-weight: 400;
+}
```

Comment:
- [ ] On some screen sizes the subtitle of your contact section is not centered on the page. Please, adjust the position of this element to fit correctly on different mobile screen sizes(NOTE: Minimum mobile screen size should be 375px. Errors in screen sizes less than 375px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size).
 -
<img width="488" alt="image" src="https://user-images.githubusercontent.com/61919617/186194544-53a55648-d9c0-499c-a97a-8a3b5e5c0d24.png">